dotCMS/src/main/java/com/dotcms/rendering/velocity/services/VTLLoader.java

```diff
@@ -0,0 +1,144 @@
+package com.dotcms.rendering.velocity.services;
+
+import com.dotmarketing.util.Config;
+import com.dotmarketing.util.ConfigUtils;
+import com.dotmarketing.util.Logger;
+import com.dotmarketing.util.UtilMethods;
+
+import java.io.BufferedInputStream;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+
+import org.apache.velocity.exception.ResourceNotFoundException;
+
+import com.liferay.util.FileUtil;
+
+public class VTLLoader implements DotLoader {
+    private static String velocityCanoncalPath;
+    private static String assetCanoncalPath;
+    private static String assetRealCanoncalPath;
+    private static String VELOCITY_ROOT;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "VELOCITY_ROOT" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3008)